### PR TITLE
doc: 6781 concat normalization v1

### PR DESCRIPTION
**Pull request by jmtaylor90**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at
   https://docs.suricata.io/en/latest/devguide/contributing/contribution-process.html
- [x] I have signed the Open Information Security Foundation contribution agreement at
   https://suricata.io/about/contribution-agreement/ (note: this is only required once)
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://redmine.openinfosecfoundation.org/issues/6781

Describe changes:
- document how duplicate header names are handled
- rtd build of this branch - https://rtd-test-suricata.readthedocs.io/en/latest/rules/http-keywords.html
-

### Provide values to any of the below to override the defaults.

To use a pull request use a branch name like `pr/N` where `N` is the
pull request number.

Alternatively, `SV_BRANCH` may also be a link to an
OISF/suricata-verify pull-request.

```
SV_REPO=
SV_BRANCH=
SU_REPO=
SU_BRANCH=
LIBHTP_REPO=
LIBHTP_BRANCH=
```

### CODE REVIEW

**Review by jufajardini — CHANGES_REQUESTED** [review]
Thanks for taking up this one (and it is indeed better to review it separately from the other HTTP doc changes :P )!

I wonder if it would make sense to have this section at the beginning of this chapter, before the HTTP keywords, as this applies to many of the existing buffers.

File: doc/userguide/rules/http-keywords.rst [hunk 1]
```diff
@@ -1091,11 +1076,6 @@ Example HTTP Request::
 .. note:: There are headers that will not be included in the ``http.header``
   buffer, specifically the :ref:`http.cookie` buffer.
 
-.. note:: If there are multiple values for the same header name, they are
-  concatenated with a comma and space (", ") between each value.
-  More information can be found in RFC 2616
-  `<https://www.rfc-editor.org/rfc/rfc2616.html#section-4.2>`_
-
```
[the remark this answers is not in the document]
Comment (jmtaylor90) [reply doc/userguide/rules/http-keywords.rst:1098]: Ah yes, will get that fixed up in the next pr

**Comment by jmtaylor90** [comment]
> Thanks for taking up this one (and it is indeed better to review it separately from the other HTTP doc changes :P )!
> 
> I wonder if it would make sense to have this section at the beginning of this chapter, before the HTTP keywords, as this applies to many of the existing buffers.

Sounds like a plan, will get things moved around. Thanks for the review!

**Comment by jmtaylor90** [comment]
continued in https://github.com/OISF/suricata/pull/10937

**Pull request closed without merging**